packages/sdk/test/unit/sdkBase.spec.ts

```diff
@@ -1118,6 +1113,39 @@ describe("NxtpSdkBase", () => {
       expect(res.transactionResponse.chainId).to.be.eq(sendingChainId);
     });
 
+    it.skip("finish transfer starts polling if not polling already, and stops after", async () => {
```

Comment:
why is this skipped if it was just added?

Reply:
skipped because needs to be deleted. modified expectations to unskip, but may still need to be deleted